src/locales/en-US/commands.json

```diff
@@ -230,7 +230,8 @@
     "github": "{{Constants.GITHUB_LOGO}} **[GitHub repository](https://github.com/SwitchbladeBot/switchblade)**"
   },
   "guildinfo": {
-    "commandDescription": "Sends information about the current server",
+    "commandDescription": "Sends information about the current server or from another guild",

```

Comment:
It might be a bit better to use „Sends information about the current or another server” or „... another guild”. I'm not entirely sure.

Reply:
guild is for api/bots, server is for interface.

Reply:
Ah, I see.